starter-code/functions-and-arrays.js

```diff
@@ -1,14 +1,25 @@
 // Find the maximum
 function maxOfTwoNumbers (first, second) {
-
+ if ( first < second) {
+   console.log(second);
+ } else {
+   console.log(first);
+ }
 }
 
 var largest = maxOfTwoNumbers(2, 6);
 console.log(largest);
 
 // Finding Longest Word
-function findLongestWord (words) {
-
+function findLongestWord (word) {
+
+ var longestWord = '';
+ words.forEach(function(word) {
```

Comment:
WATCH OUT! You are using the variable words (outside the function) instead of the parameter of the function (word, at line 14). You will see it clearer in the next comment.